Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```diff
@@ -30,7 +30,8 @@
         amount: 1
       - id: PlushieAtmosian
         amount: 1
-
+      - id: PlushieDiona
+        amount: 1
```

Comment:
this isn't your fault, but you can remove the `amount: 1` from every single entry here. The EntitySpawnEntry struct defaults the amount to 1.

Reply:
Done